src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/tableView.js

```diff
@@ -0,0 +1,249 @@
+class TableViewVisualization extends Visualization {
+    static inputType = "Any"
+
+    onDataReceived(data) {
+
+        var tableOf = function (content, level) {
+            var open = '<table class="level' + level + '">';
+            return open + content + "</table>";
+        }
+
+        var hasExactlyKeys = function (keys, obj) {
+            return Object.keys(obj).length == keys.length && keys.every(k => obj.hasOwnProperty(k));
+        };
+
+        var getAtNestedKey = function (data, key) {
```

Comment:
why not `function getAtNestedKey(data,key)` instead?

Reply:
Ask @kustosz :D